docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml

```diff
@@ -8309,10 +8581,11 @@ ecl getwuid -n MyCommonJobName --limit=100</programlisting>
                 </row>
 
                 <row>
-                  <entry>--wait-read=&lt;Secs&gt;</entry>
+                  <entry>--noKeepAlive </entry>
```

Comment:
It looks like the wait-read entry was deleted by mistake.

This is one reason why copy and paste is a problem...

Reply:
@ghalliday I agree that redundant text and copy/paste are issues--we have a solution we will be using soon. 
As for this PR, the wait-read option is still included in every list of options where it is needed (37 times). This one is on line 8577.

Reply:
It appears like wait-read section was a duplicate ?

Reply:
The deleted item had been a duplicate and needed removsal